x/ibc/03-connection/types/query.go

```diff
@@ -0,0 +1,66 @@
+package types
+
```

Comment:
renamed to `query.go` + added necessary response constructors. not sure why github diff didn't recognize this properly